lib/js-api/src/api/content-rest-api/docs/SizeDetails.md

```diff
@@ -0,0 +1,12 @@
+# Size Details
+
+## Properties
+
+ Name              | Type   | Description                                                                 | Notes             
+-------------------|--------|-----------------------------------------------------------------------------|-------------------
```

Comment:
Getting this issue for that table: 
<img width="1136" alt="image" src="https://github.com/user-attachments/assets/c1831725-4ae8-46da-b47a-197a012b318f">

Reply:
Since this file is removed, and its data moved to NodesApi.md file, this warning can be ignored